builtin/settingtypes.txt

```diff
@@ -430,40 +583,41 @@ show_debug (Show debug info) bool false
 
 [Server / Singleplayer]
 
-#    Name of the server.
+#    Name of the server, to be displayed when players join and in the serverlist.
 server_name (Server name) string Minetest server
 
-#    Description of server.
+#    Description of server, to be displayed when players join and in the serverlist.
 server_description (Server description) string mine here
 
-#    Domain name of server.
+#    Domain name of server, to be displayed in the serverlist.
 server_address (Server address) string game.minetest.net
 
-#    Homepage of server.
+#    Homepage of server, to be displayed in the serverlist.
 server_url (Server URL) string http://minetest.net
 
-#    Automaticaly report to masterserver.
+#    Automaticaly report to the serverlist.
 server_announce (Announce server) bool false
 
-#    Announce to this masterserver.
-#    If you want to announce your ipv6 address - use  serverlist_url = v6.servers.minetest.net.
+#    Announce to this serverlist.
+#    If you want to announce your ipv6 address, use  serverlist_url = v6.servers.minetest.net.
 serverlist_url (Serverlist URL) string servers.minetest.net
 
 [*Network]
 
 #    Network port to listen (UDP).
+#    This value will be overridden when starting from the main menu.
 port (Server port) int 30000
 
 #    The network interface that the server listens on.
 bind_address (Bind address) string
 
-#    Set to true to disallow old clients from connecting.
+#    Set to true to disallow old, possibly not fully compatible clients from connecting.
```

Comment:
What does this have to do with "full compatibility"? If we support a network protocol version, we _are_ fully compatible to it.